packages/react-ui-map/src/Mapbox/Mapbox.js

```diff
@@ -28,36 +26,39 @@ export default class Mapbox extends PureComponent {
 
   constructor(props, context) {
     super(props, context)
+    mapboxgl.accessToken = props.token
 
     this.state = {
       map: null,
     }
   }
 
   getChildContext() {
-    return { map: this.state.map }
+    return {
+      map: this.state.map,
+    }
   }
 
   componentDidMount() {
-    mapboxgl.accessToken = this.props.token
-    const map = new mapboxgl.Map({
-      container: this.container,
-      style: this.props.style,
-      center: this.props.center,
-      zoom: this.props.zoom,
-      theme: this.props.theme,
-    })
-
-    map.on('load', () => {
-      this.setState({ map })
-    })
+    if (this.props.center) {
+      const map = new mapboxgl.Map({
+        container: this.container,
+        style: this.props.style,
+        center: this.props.center,
+        zoom: this.props.zoom,
+        theme: this.props.theme,
+      })
+      map.on('load', () => {
+        this.setState({ map })
+      })
+    }
   }
 
   componentWillReceiveProps(nextProps) {
     const { map } = this.state
 
-    if (nextProps.boundingBox) {
-      fly(map, nextProps.boundingBox)
+    if (map && nextProps.center) {
```

Comment:
Don't you want to check if current `center` prop is different from the next `center` prop?  I personally think this is a little much considering this component is supposed to be generic. I don't think we should be enforcing a `flyTo` at all.  Maybe adding a prop that toggles whether or not we do this?
```
flyOnCenterChange: false
``` 

@aribouius thoughts?

Reply:
Luke please explain the rationale for this change.

Reply:
we we don't get a bounding box from any of our data endpoints. we just get a center or centroid for the city. 

if you really want to center in on a bounding box you just call `map.fitbounds(bbox)` which is the same as saying `map.setCenter(bbox.getCenter())` with some adjustments for zooming and padding.

Reply:
for our usecase, it works like this:

render map with center -> get results -> fitbounds on those results

then

user changes location -> get new results -> fit bounds on those results. 

i like the idea of checking the center change and toggling fly behavior.